### PR TITLE
Adding apiserver pool

### PR DESCRIPTION
**Pull request by l337ch**

### CODE REVIEW

**Comment by l337ch** [comment]
this has been tested with apiserver pool size of 5 and node count of 20. i haven't tested a larger cluster yet but will try in the morning.

**Comment by l337ch** [comment]
retest please

**Comment by l337ch** [comment]
retest this please

**Comment by l337ch** [comment]
retest this please

**Comment by l337ch** [comment]
retest this please

**Comment by l337ch** [comment]
retest this please

**Comment by l337ch** [comment]
retest this please

**Comment by maratoid** [comment]
retest this please

**Comment by maratoid** [comment]
retest this please

**Comment by l337ch** [comment]
i need to rebase this and fix all the merge conflicts. i will also turn up the number of connections that nginx can handle

**Comment by l337ch** [comment]
This is ready for review now

File: ansible/roles/kubernetes/templates/hyperkube-docker-apiserver.service.ansible [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  {{hyperkube_image}} \
+    /hyperkube \
+    kubelet \
+    --cluster_dns={{dns_ip}} \
+    --cluster_domain={{dns_domain}} \
+    --config=/etc/kubernetes/manifests-override \
+    --enable_server \
+    --hostname_override={{master_private_ip}} \
+    --v={{kubelet_v}}
```
Comment (spiffxp) [inline ansible/roles/kubernetes/templates/hyperkube-docker-apiserver.service.ansible]: `kubelet_v` doesn't exist anymore; workaround is just use `kubelet.v`, or convert this to use the same `.iteritems()` syntax the other hyperkube units are using

**Comment by spiffxp** [comment]
LGTM aside from that one typo

File: ansible/roles/kubernetes/templates/hyperkube-docker-apiserver.service.ansible (hunk 1 above)
Comment (l337ch) [inline ansible/roles/kubernetes/templates/hyperkube-docker-apiserver.service.ansible]: i'll update that. thx. i have another PR coming soon anyways. adding nginx status page

**Comment by l337ch** [comment]
retest this please

**Comment by l337ch** [comment]
After talking to @maratoid.i will for his PR and rebase master when it's merged.

**Comment by l337ch** [comment]
master has been merged

**Comment by l337ch** [comment]
retest this please

**Comment by maratoid** [comment]
retest this please

**Comment by l337ch** [comment]
retest this please

**Comment by l337ch** [comment]
@maratoid i think there's some other resources that needs to be resurrected?
'''
* 1 error(s) occurred:

* aws_vpc_dhcp_options.vpc_dhcp: Error retrieving DHCP Options: InvalidDhcpOptionID.NotFound: The dhcpOption ID 'dopt-4e936c2b' does not exist
	status code: 400, request id: 
* 1 error(s) occurred:

* aws_network_acl.vpc_acl: InvalidNetworkAclID.NotFound: The networkAcl ID 'acl-15c3ba70' does not exist
	status code: 400, request id: [0m[0m
'''

**Comment by maratoid** [comment]
let me see about making sure everything is cleaned up here.

**Comment by maratoid** [comment]
retest this please

**Comment by maratoid** [comment]
retest this please

**Pull request merged**